src/refline/srccheck/pyflakes.py

```diff
@@ -5,13 +5,14 @@
 """
 Implementation of the command-line I{pyflakes} tool.
 """
-
+from __future__ import print_function
```

Comment:
This is probably worth a changelog note.

(How did version 1.0 claim "official Python 3 support" with this kind of SyntaxError lurking it the source??)

Reply:
okay.